tests/frameworks/langchain/test_model_server.py

```diff
@@ -0,0 +1,352 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import os
+import pathlib
+import subprocess
+from subprocess import PIPE, Popen
+
+import pytest
+
+import mlrun
+
+relative_asset_path = "mlrun/frameworks/langchain/model_server.py"
+langchain_model_server_path = str(
+    pathlib.Path(__file__).absolute().parent.parent.parent.parent / relative_asset_path
+)
+
+
+#: if true, delete the model after the test
+_OLLAMA_DELETE_MODEL_POST_TEST = False
+_OLLAMA_MODEL = "qwen:0.5b"
+_OPENAI_MODEL = "gpt-3.5-turbo-instruct"
+_HUGGINGFACE_MODEL = "gpt2"
+
+
+# To run this test, you need to:
+# 1. install ollama on your computer
+# 2. pull the desired model from the ollama repository (for example, in terminal: ollama pull llama3)
+# 3. run ollama with said model in terminal (for example, in terminal: ollama run llama3)
+def ollama_check_skip():
+    """
+    Check if ollama is installed
+    """
+    try:
+        result = subprocess.run(["ollama", "--help"], stdout=PIPE)
+    except Exception as e:
+        print(f"Error checking ollama: {e}")
```

Comment:
why would it be raised?

Reply:
didn't find command ollama (or something similar)

Reply:
ah so True is to skip, ok!